### ISSUE TITLE
Does it support causal mask?

### ISSUE THREAD

**Issue by shaoyucheng** [philipperemy/keras-attention#67]
The effect is the same as the argument "use_casual_mask" in keras official layer class Attention.
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Attention
Thanks for your useful module and expect your reply.

**Comment by shaoyucheng** [comment]
> I don't think there is at the moment.

got it, thanks!
